docs/Growing.md

```diff
@@ -117,6 +117,139 @@ functionality and then moves on to producing code that passes these tests. There
 having good test coverage and code being designed for testability, which will in turn often prevent future changes from
 breaking old code. Much of mojo.js was developed using TDD.
 
+## Prototype
```

Comment:
For consistency second level headings should be upper case.
```suggestion
## PROTOTYPE
```

Reply:
Actually, the all uppercase Perl convention is not so nice for Markdown.